ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c

```diff
@@ -3111,7 +3155,8 @@ process_vlv_index(backentry *ep, ImportWorkerInfo *info)
         struct vlvIndex *vlv_index = ps->vlv_index;
         Slapi_PBlock *pb = slapi_pblock_new();
         slapi_pblock_set(pb, SLAPI_BACKEND, be);
-        if (vlv_index && (ctx->indexAttrs==NULL || attr_in_list(vlv_index->vlv_name, ctx->indexAttrs))) {
+        if (vlv_index && vlv_index->vlv_attrinfo &&
+            is_reindexed_attr(vlv_index->vlv_attrinfo->ai_type , ctx, ctx->indexVlvs)) {
```

Comment:
Any reason why relaying on vlv_index->vlv_attrinfo->ai_type rather than vlv_index->vlv_name

Reply:
Yes (BTW: I fall in that pitfall too! (I first used vlvName then got in trouble)  :wink: 
That is because in truncate_index_dbi (that deletes the vlv index that are reindexed) the vlv_index is not available but only the ai.